pages/_app.tsx

```diff
@@ -1,9 +1,11 @@
 import * as React from 'react';
+import dynamic from 'next/dynamic';
 import { AppProps } from 'next/app';
 import Head from 'next/head';
 
 import '../styles/globals.css';
-import HeaderMobile from 'components/Layout/HeaderMobile';
+
+const DynamicHeader = dynamic(() => import('components/Header'), { ssr: false });

```

Comment:
Wouldnt it be nice for SEO, if we render any header as default and change the header at mount in the browser?